sunbeam-python/sunbeam/steps/upgrades/intra_channel.py

```diff
@@ -160,6 +165,18 @@ def get_plan(self) -> list[BaseStep]:
             ]
         )
 
+        if is_maas_deployment(self.deployment):
```

Comment:
We'll need to better factor in the whole refresh depending on being a manual or maas deployment, but it's fine for the moment, since we need the fix urgently.